airflow/configuration.py

```diff
@@ -525,12 +536,12 @@ def _get_env_var_option(self, section, key):
     def _get_cmd_option(self, section, key):
         fallback_key = key + '_cmd'
         # if this is a valid command key...
-        if (section, key) in AirflowConfigParser.as_command_stdout:
+        if (section, key) in self.as_command_stdout:
             # if the original key is present, return it no matter what
-            if self.has_option(section, key):
+            if ConfigParser.has_option(self, section, key):
```

Comment:
I override `has_option` function below.
`has_option` needs to check the config is in `default_config` and `self`.

if it use `self.has_option`, it'll look into default option which contains default `sql_alchemy_conn` so It'll never override `_cmd`.